cics-bundle-maven-plugin/pom.xml

```diff
@@ -169,14 +170,31 @@
         <groupId>org.sonatype.plugins</groupId>
         <artifactId>nexus-staging-maven-plugin</artifactId>
       </plugin>
+      <plugin>
```

Comment:
Shouldn't these all be in pluginManagement if they're just forcing the version?

Reply:
Oops I thought I did add them to pluginManagement, must have missed this one. Will update